uitools/toolkitwidgets.pri

```diff
@@ -13,6 +13,8 @@
 # limitations under the License.
 include($$PWD/common.pri)
 
+QT += widgets webenginewidgets svg
```

Comment:
❓ 
Is `webenginewidgets` new? Seems right we'd need it.

Reply:
It's been around since Qt 5.4 AFAIK. The inclusion of "OAuthView" in Widgets forces it as a requirement unless we can make it a conditional compilation in the future.